EDJE_*_PRG assumes a program with $EXEEXT, but edje_recc is a script

--- origsrc/edje-1.0.1/src/bin/Makefile.am	2011-04-19 01:50:23.000000000 -0500
+++ src/edje-1.0.1/src/bin/Makefile.am	2011-11-03 21:34:40.708868800 -0500
@@ -1,7 +1,7 @@
 
 MAINTAINERCLEANFILES = Makefile.in
 
-bin_SCRIPTS = @EDJE_RECC_PRG@
+bin_SCRIPTS = edje_recc
 
 bin_PROGRAMS = @EDJE_CC_PRG@ @EDJE_DECC_PRG@ @EDJE_PLAYER_PRG@ @EDJE_INSPECTOR_PRG@ @EDJE_EXTERNAL_INSPECTOR_PRG@
 
